### PR TITLE
Remove incorrect information on ignition files

### PR DESCRIPTION
**Pull request by rh-max**
@jboxman Could you PTAL and merge & cherry-pick to `enterprise-4.2` and `enterprise-4.3`?

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@rh-max, have you gotten this approved by the multiarch QE team?

**Comment by rh-max** [comment]
@kalexand-rh By the QE team I didn't, but I did by a multiarch engineer.

**Comment by kalexand-rh** [comment]
Will you get Barry to approve it, if he hasn't already?

**Comment by rh-max** [comment]
@kalexand-rh Is Barry a QE? What's his full name or nickname, please?

**Comment by kalexand-rh** [comment]
@rh-max, Barry Donahue's provided QE acks for the Z installation docs before. He's usually easier to reach through email than GitHub.

**Comment by rh-max** [comment]
@kalexand-rh Requested, thank you.

File: modules/installation-ibm-z-user-infra-machines-iso.adoc [hunk 1]
```diff
@@ -47,7 +47,7 @@ Download the following files:
 ... The machine host and domain name in the form `hostname.domainname`. Omit this value to let {op-system} decide set it.
 ... The network interface name. Omit this value to let {op-system} decide set it.
 ... If you use static IP addresses, an empty string.
-** For `coreos.inst.ignition_url=`, specify the Ignition file for the machine role. The bootstrap machine ignition file is called `bootstrap-0`, the master ignition files are numbered 0 through 2, the worker ignition files from 0 upwards. All other parameters can stay as they are.
+** For `coreos.inst.ignition_url=`, specify the Ignition file for the machine role. The bootstrap machine ignition file is called `bootstrap-0`. All other parameters can stay as they are.
```
Comment (dbenoit17) [inline modules/installation-ibm-z-user-infra-machines-iso.adoc]: I agree a change should be made here, as the original wording combes two distinct ideas.  Could we change this to be more along the lines of:

```
** For `coreos.inst.ignition_url=`, specify the Ignition file for the machine role (bootstrap.ign, master.ign, or worker.ign). 

** All other parameters can stay as they are.
```

**Review by dbenoit17 — CHANGES_REQUESTED** [review]

File: modules/installation-ibm-z-user-infra-machines-iso.adoc (hunk 1 above)
Comment (rh-max) [reply modules/installation-ibm-z-user-infra-machines-iso.adoc]: @dbenoit17 Thanks David. Does it look good now?
https://coreos-ignition--ocpdocs.netlify.com/openshift-enterprise/latest/installing/installing_ibm_z/installing-ibm-z.html#installation-user-infra-machines-iso-ibm-z_installing-ibm-z
If yes, I will ask for this to be merged.

**Review by dbenoit17 — APPROVED** [review]
lgtm

**Comment by rh-max** [comment]
@dbenoit17 Thank you.
@kalexand-rh If it looks good now, could you please merge & cherry-pick to `enterprise-4.2` and `enterprise-4.3`?

File: modules/installation-ibm-z-user-infra-machines-iso.adoc [hunk 2]
```diff
@@ -47,7 +47,8 @@ Download the following files:
 ... The machine host and domain name in the form `hostname.domainname`. Omit this value to let {op-system} decide set it.
 ... The network interface name. Omit this value to let {op-system} decide set it.
 ... If you use static IP addresses, an empty string.
-** For `coreos.inst.ignition_url=`, specify the Ignition file for the machine role. The bootstrap machine ignition file is called `bootstrap-0`, the master ignition files are numbered 0 through 2, the worker ignition files from 0 upwards. All other parameters can stay as they are.
+** For `coreos.inst.ignition_url=`, specify the Ignition file for the machine role (`bootstrap.ign`, `master.ign`, or `worker.ign`).
```
Comment (kalexand-rh) [inline modules/installation-ibm-z-user-infra-machines-iso.adoc:50]: I'd edit this to remove the parentheses.

Please squash your commits.

Comment (kalexand-rh) [reply modules/installation-ibm-z-user-infra-machines-iso.adoc:50]: @rh-max, I have not seen that before! But GitHub had an outage yesterday, so please try rebasing and force pushing your local branch again before you make a new PR.

Comment (rh-max) [reply modules/installation-ibm-z-user-infra-machines-iso.adoc]: @kalexand-rh Rebasing worked. Please let me know if anything else is missing.

**Comment by rh-max** [comment]
@kalexand-rh Thanks!

**Pull request merged**